test3.tf

```diff
@@ -58,6 +58,7 @@ resource "azurerm_storage_account" "example" {
       retention_policy_days = 10
     }
   }
+  enable_https_traffic_only = true
 }
 
 resource "azurerm_sql_server" "example" {
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure that Azure Active Directory Admin is configured'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |

 <!-- bbc6043f-5928-4653-bfc7-41349a13f9b6 -->